### PR TITLE
Make QCEngine tolerant of `cpuinfo` failure to populate `brand_raw`, `brand`

### PR DESCRIPTION
**Pull request by dotsdl**
## Description

Currently, we observe frequent failure of `cpuinfo` to give brand information when used within Docker containers on Pacific Research Platform. It is unclear why, but making QCEngine tolerant of this information being missing would help in production usage.

## Changelog description

QCEngine calls now tolerant of missing `brand_raw` or `brand` info from py-cpuinfo.

## Status

- [x] Code base linted
- [x] Ready to go

### CODE REVIEW

File: qcengine/config.py
Comment (WardLT) [inline qcengine/config.py:56]: The nested `try/except` blocks seem a bit unnecessary. Would you mind changing to a `.get('brand', 'unknown')` syntax here so we store `cpu_brand='unknown'` if the resolution fails twice?

File: qcengine/config.py
Comment (WardLT) [inline qcengine/config.py:64]: I'd keep this line as `[key]` so we get `KeyError`s if a key is unknown still. I'm not sure what returning `None` will do to other parts of QCEngine.

**Review by WardLT — CHANGES_REQUESTED** [review]

**Comment by dotsdl** [comment]
Thanks @WardLT for the review! I've made the changes you requested. Let me know if there's anything else I can do to move this forward!

**Comment by dotsdl** [comment]
Looks like the remaining CI errors may be unrelated to the changes in this PR. @loriab is this consistent with your view?

**Comment by loriab** [comment]
Agreed, if you make this change https://github.com/MolSSI/QCEngine/pull/338/files#diff-d66842b418a1c03f41438563803c987c9defd1811ea0af131e285ba540c8c0d6R208, it'll fix it. May as well get that PR approved and merged, actually. With that change and this, can do a new minor release instead of just the patch. It'll help qcng versions catch up to qcel, anyways.

**Comment by dotsdl** [comment]
Thanks @loriab! I'll review and merge #338, then pull that in to this PR today. Definitely appreciate getting a minor release out; will be a huge help for us in prod use!

File: qcengine/config.py
Comment (loriab) [inline qcengine/config.py:56]: not a strong suggestion -- just wondering if `(none)` or `(unknown)` would be more consistent with other "missing" schema values (like `model.basis = "(none)"`)

Comment (WardLT) [reply qcengine/config.py:56]: I'm all for simplifying the vocabulary. `(unknown)` would be my vote (the CPU has a brand, we just don't know it).

**Review by WardLT — APPROVED** [review]
Last change would be a minor, optional revision.

Thanks!

**Comment by WardLT** [comment]
I just realized that I had the power to make that change, so I did so. 

I'll merge after we merge #338, just to be careful about merging with passing tests and such

**Comment by dotsdl** [comment]
Thank you @WardLT! #338 is now merged, and I've merged `master` here to get those changes. Waiting on the tests now.

**Pull request merged**
